3-0-libraries.md

```diff
@@ -248,14 +248,73 @@ summed when displayed on this page.
+
+## What if I assign a library to the wrong parent aliquot?
+Please email gsi@oicr.on.ca or file a JIRA ticket in GSI Common to get assistance from the MISO team.
+
+## What if I forget to put a library dilution in a pool?
+If the library dilution has not been created:
+
+1. From _Tracking_, _Library_, select the library that needs to be added.
+1. In the _Library Dilution_ section, hover over _Options_ and select _Add Library Dilution_.
+1. Fill out the new row in the table, then click _Add_.
+
+Once the dilution exists:
+
+1. From _Tracking_, _Pool_, select the pool that needs the additional library dilution.
+1. In the _Pool Elements_ section, find the second table, labelled _Select poolable elements_.
+1. Enter the library name, library alias, or dilution name in the _Search_ box.
+1. Find the correct dilution in the list and click the plus button beside it.
+
+## How do I change the targeted sequencing type on a library?
+Targeted sequencing is connected to the dilution since the same library can be used for multiple targeted sequencing panels.
+
+1. From _Tracking_, _Library_, select the library that needs to be changed.
+1. In the _Library Dilution_ section, under the _Edit_ column, click the pencil beside the dilution that needs to be changed.
+1. Select a new _Targeted Sequencing_ from the list.
+1. Under the _Edit_ column, click _Save_.
+
+## How can I add a new targeted sequencing type, kit, or anything else in drop-down menus?
+For targeted sequencing and indices, please email gsi@oicr.on.ca or file a JIRA ticket in GSI Common to get assistance from the MISO team.
+
+Kits can be added easily:
+
+1. From _Tracking_, _Kits_, select the type of kit you want to add.
+1. Click _Create Kit Descriptor_.
+1. Fill in the form. The _Stock Level_ is not currently used, so leave it at zero.
+1. Carefully select the _Kit Type_ and _Platform_. This will decide where the kit is displayed.
+1. Click _Save_.
+
+## How do I make bulk orders?
+Alas, this is not possible yet.
+
+## How are matrix tube barcodes assigned to tubes?
+Barcodes can be assigned on an individual edit page or in bulk.
+
+To change a single library:
+
+1. From _Tracking_, _Library_, select the library that needs to be changed.
+1. In the top right, you will see a box labelled _Barcode_.
+1. Hover over the down arrow and select _Update Barcode_.
+1. Scan the barcode.
+1. Click _Save_.
+
+To change many libraries:
+
+1. From _Tracking_, _Library_, check the box beside each library that needs to be changed.
+1. From _Bulk actions_, choose _Update selected_ and click _Go_.
+1. Select the cell in the _Matrix Barcode_ column for the library and scan the barcode.
+1. Repeat for all the libraries.
+1. Click _Save_.
+
+## What is the importance of selecting a study since it prevents the Sequencing stage from adding pools to the lane containers?
+The study is important for submitting to the ERA. Although we are not using that feature of MISO, there's no easy way to disable it.
```

Comment:
The wording of the question has me think that the user believes that adding a study to a project makes it impossible to add the pool to a container. Please update the answer to address that possible interpretation.